cc3d/core/Configuration/SettingUtils.py

```diff
@@ -212,3 +231,51 @@ def synchronize_global_and_default_settings(default_settings, global_settings, g
 
 def synchronizeGlobalAndDefaultSettings(default_settings, global_settings, global_settings_path):
     return synchronize_global_and_default_settings(default_settings, global_settings, global_settings_path)
+
+
+def _handle_settings_value_xml(_val: str):
+    """
+    Handles data intended for sqlite but used in python types
+    """
+    if _val == 'True':
+        return '1'
+    elif _val == 'False':
+        return '0'
+    return _val
+
+
+def default_settings_dict_xml() -> Dict[str, Any]:
+    """
+    Returns default data dictionary
+    """
+    fp = _default_setting_path_xml()
+    if not os.path.isfile(fp):
```

Comment:
@maciekswat said something about wanting to use pathlib instead of the OS module, but I forgot the reason.

Reply:
unless you "normalize" paths - meaning use forward or reverse slashes depending on the platform the os may mix forward and reverse slashes. pathlib handles it much better.  In general, pathlib gives you more path-centric  access to all file operations. with os modules, you need a lot more modules to handle certain operations e.g. making directly

Reply:
I don't see pathlib being used in this neighborhood of the code, so can we reserve pathlib utlilization for a separate PR?

Reply:
Definitely. I have done this previously.